libsrc/osic1p/bootstrap.s

```diff
@@ -45,6 +45,8 @@ GETCHAR         :=      $FFBF           ; gets one character from ACIA
 FIRSTVISC       =       $85             ; Offset of first visible character in video RAM
 LINEDIST        =       $20             ; Offset in video RAM between two lines
 
+        lda     #0
+        sta     load
```

Comment:
You found the bug; and, you recognized what needs to be fixed.  I think that you should get the credit, in the project's logs.  Therefore, you should submit the Pull Request.

(You can close this one after Oliver merges the PR that you send to him.)